src/about.html

```diff
@@ -90,7 +90,11 @@ <h2 id="contact">Contact us <a href="http://twitter.com/tosdr" class="btn btn-bi
 			<section id="team">
 				<div class="page-header"><h2>The team <a href="blog/" class="btn btn-big btn-primary">Read the blog »</a></h2>
 				</div>
+<<<<<<< HEAD
+				
+=======
 
+>>>>>>> 43bc4f6ea01b4176832313d8e85b4fd3abe49206
```

Comment:
This is a merge conflict. Try the following instead:

````
git checkout master
git pull
git branch chris-twitter-handle
git checkout chris-twitter-handle
# do your editing
git commit -am"Change Twitter handle Chris"
git push -u origin chris-twitter-handle
````

Then you can go to the github website and create a new pull request from the `chris-twitter-handle` branch.